app/mailers/password_reset_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class PasswordResetMailer < ActionMailer::Base
+  # Input:
+  #   user: chef object returned by server
+  def password_reset(user)
+    @user = user
```

Comment:
Because instance variables can be used when rendering the template; see https://github.com/chef/oc-id/pull/40/files#diff-b23ea7309a5154e9426ac928b8a5e99eR1.